src/Routes/applicationRoutes.ts

```diff
@@ -52,7 +52,7 @@ applicationRoutes.get(
     const { id } = req.params;
 
     try {
-      const application = await ApplicationModel.findById(id, { __v: 0 })
+      const application = await MatchModel.findById(id, { __v: 0 })
```

Comment:
```suggestion
      const match = await MatchModel.findById(id, { __v: 0 })
```
Will need to do this for vars at some point.